cloudflare/resource_cloudflare_load_balancer.go

```diff
@@ -165,6 +172,7 @@ func resourceCloudflareLoadBalancerCreate(d *schema.ResourceData, meta interface
 		Proxied:        d.Get("proxied").(bool),
 		TTL:            d.Get("ttl").(int),
 		SteeringPolicy: d.Get("steering_policy").(string),
+		Persistence:    d.Get("session_affinity").(string),
```

Comment:
Something that we might have a discussion about in `cloudflare/cloudflare-go` is the fact that we're using `Persistence` on `cloudflare.LoadBalancer` despite the API response using `session_affinity`. While not wrong, it does seems like a potential for misunderstanding.